manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+        radius_line_frequency=1,
+        azimuth_units="PI radians",
+        azimuth_offset=None,
+        azimuth_direction=None,
+        azimuth_buff=SMALL_BUFF,
+        radius_config={"label_direction": DOWN + LEFT},
+        radius_max=None,
+        **kwargs,
+    ):
+        self.azimuth_units = azimuth_units
+        self.azimuth_frequency = (
+            {
+                "PI radians": (1 / 20),
+                "TAU radians": (1 / 20),
+                "degrees": (1 / 36),
+                "gradians": (1 / 40),
+                None: 1,
+            }[self.azimuth_units]
+            if azimuth_line_frequency is None
+            else azimuth_line_frequency
+        )
+        self.azimuth_offset = (
+            {
+                "PI radians": 0,
+                "TAU radians": 0,
+                "degrees": PI / 2,
+                "gradians": PI / 2,
+                None: 0,
+            }[self.azimuth_units]
+            if azimuth_offset is None
+            else azimuth_offset
+        )
+        self.azimuth_direction = (
+            {
+                "PI radians": "CCW",
+                "TAU radians": "CCW",
+                "degrees": "CW",
+                "gradians": "CW",
+                None: "CCW",
+            }[self.azimuth_units]
+            if azimuth_direction is None
+            else azimuth_direction
+        )
+        super().__init__(
+            color=color,
+            x_line_frequency=self.azimuth_frequency * TAU,
+            y_line_frequency=radius_line_frequency,
+            x_axis_config=radius_config,
+            x_min=None if (radius_max is None) else -radius_max,
+            x_max=radius_max,
+            y_min=-config["frame_x_radius"] if (radius_max is None) else -radius_max,
+            y_max=config["frame_x_radius"] if (radius_max is None) else radius_max,
+            **kwargs,
+        )
+        self.azimuth_buff = azimuth_buff
+        self.prepare_for_nonlinear_transform()
+        self.background_lines.apply_function(
+            lambda p: np.array([p[0] * np.sin(p[1]), p[0] * np.cos(p[1]), 0])
+        )
+
+    def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        for mob in self.background_lines.family_members_with_points():
+            num_curves = mob.get_num_curves()
+            if num_inserted_curves > num_curves:
+                mob.insert_n_curves(num_inserted_curves - num_curves)
+        return self
+
+    def polar_to_point(self, radius, azimuth):
+        """Gets a point from polar coordinates.
+
+        Parameters
+        ----------
+        radius : :class:`float`
+            The coordinate radius (:math:`r`).
+
+        azimuth : :class:`float`
+            The coordinate azimuth (:math:`\\theta`).
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+            The point.
+        """
+        return self.coords_to_point(radius * np.cos(azimuth), radius * np.sin(azimuth))
+
+    def pr2pt(self, radius, azimuth):
```

Comment:
Personal opinion: I think these abbreviated methods are mainly confusing (although it's certainly not as extreme as with `Text` here) and they should not be added. Further opinions are welcome.

Reply:
This is based on the template of the existing class `ComplexPlane` which includes abbreviated methods like this. Are they planned to be removed?